src/cmd/build.js

```diff
@@ -131,7 +131,8 @@ async function defaultPackageCreator({
   await eventToPromise(stream, 'close');
 
   if (showReadyMessage) {
-    log.info(`Your web extension is ready: ${extensionPath}`);
```

Comment:
so, the expected behavior here is that:
  * "web-ext build" says "it's ready"
  *  Firefox accepts the .zip file even if there's no signature nor id, as long as signature check is disabled
right?